Yosemite/YosemiteTests/Mockups/Networking/Remote/MockProductVariationsRemote.swift

```diff
@@ -0,0 +1,46 @@
+
+import Foundation
+import Networking
+
+import XCTest
+
+/// Mock for `MockProductVariationsRemote`.
+///
+final class MockProductVariationsRemote {
+    private struct ResultKey: Hashable {
+        let siteID: Int64
+        let productID: Int64
+        let productVariationIDs: [Int64]
+    }
+
+    /// The results to return based on the given arguments in `updateProductVariation`
+    private var productVariationUpdateResults = [ResultKey: Result<ProductVariation, Error>]()
+
+    /// Set the value passed to the `completion` block if `updateProductVariation()` is called.
+    ///
+    func whenUpdatingProduct(siteID: Int64, productID: Int64, productVariationID: Int64, thenReturn result: Result<ProductVariation, Error>) {
```

Comment:
We probably want this to be `whenUpdatingProductVariation`.

Reply:
yeaa 😅  updated in https://github.com/woocommerce/woocommerce-ios/pull/2539/commits/0a372c05c4034812cefeb3ef6395e2d3df225cd5